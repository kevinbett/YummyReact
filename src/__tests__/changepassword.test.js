import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import ChangePassword from '../components/user/Change_password';

describe('Change Password Component', () => {
    const wrapper = shallow(<ChangePassword />);
    it('renders without crashing', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});