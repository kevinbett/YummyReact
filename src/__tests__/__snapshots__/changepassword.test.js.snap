// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Change Password Component renders without crashing 1`] = `
<div>
  <h3
    style={
      Object {
        "marginTop": 20,
      }
    }
  >
     Change Password
  </h3>
  <form>
    <TextField
      disabled={false}
      floatingLabelFixed={false}
      floatingLabelText="New password"
      fullWidth={false}
      hintText="Enter your new password"
      multiLine={false}
      onChange={[Function]}
      rows={1}
      type="password"
      underlineShow={true}
    />
    <br />
    <RaisedButton
      disabled={false}
      fullWidth={false}
      label="Submit"
      labelPosition="after"
      onClick={[Function]}
      primary={false}
      secondary={true}
      type="submit"
    />
  </form>
</div>
`;
